### DISCUSSION TITLE
Why support never reply in 24h ??

### DISCUSSION THREAD

**Discussion by IVEN2024** [ant-media/Ant-Media-Server#6670] [General]
Why support never reply in 24h ??
we have server crash every 3 to 4 days , we send many logs no one talked to us ???

**Comment by IVEN2024** [comment]
[vertx-blocked-thread-checker] WARN  i.v.core.impl.BlockedThreadChecker - Thread Thread[vert.x-eventloop-thread-0,5,main] has been blocked for 7583 ms, time limit is 2000 ms

**Comment by muratugureminoglu** [comment]
Hi @IVEN2024 

Crash logs sent by the system sometimes delayed responses due to high overload on our side, as investigating these takes some time. Additionally, crash logs are evaluated at a different support level. 

My suggestion to you is: You have an Enterprise license, so if you report this issue to us as a ticket with a detailed reproduce scenario (with log files), you will receive a response within 24 hours. Sounds good ? 

Thanks.
